README.md

```diff
@@ -49,7 +49,6 @@ clear to read and to maintain.
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-
```

Comment:
I think this, and the other formatting change might have been automatic, I didn't do them intentionally but I can try to revert those changes if necessary.